### PR TITLE
Adding Federal University of Juiz de Fora

### PR DESCRIPTION
**Pull request by felcaetano**
We've been happily using BigBlueButton since 0.8 :)

### CODE REVIEW

File: adopters.geojson [hunk 1]
```diff
@@ -35,6 +35,8 @@
     {"type": "Feature", "properties": {"institution": "University of Duisburg-Essen, Campus Duisburg"}, "geometry": {"type": "Point", "coordinates": [6.8008580326401, 51.4290899]}},
     {"type": "Feature", "properties": {"institution": "Hamburg University, MIN Faculty"}, "geometry": {"type": "Point", "coordinates": [9.9835937, 53.5578783]}},
     {"type": "Feature", "properties": {"institution": "Chemnitz University of Technology"}, "geometry": {"type": "Point", "coordinates": [12.92812, 50.83889]}},
+    {"type": "Feature", "properties": {"institution": "Universidade Federal de Juiz de Fora - Campus sede"}, "geometry": {"type": "Point", "coordinates": [-21.77648953804816, -43.36877001614874]}},
```
Comment (tibroc) [inline adopters.geojson]:
```suggestion
    {"type": "Feature", "properties": {"institution": "Universidade Federal de Juiz de Fora - Campus sede"}, "geometry": {"type": "Point", "coordinates": [-43.36877001614874, -21.77648953804816]}},
```

File: adopters.geojson [hunk 2]
```diff
@@ -35,6 +35,8 @@
...
+    {"type": "Feature", "properties": {"institution": "Universidade Federal de Juiz de Fora - Campus GV"}, "geometry": {"type": "Point", "coordinates": [-18.856339060042096, -41.94918019310874]}},
```
Comment (tibroc) [inline adopters.geojson]:
```suggestion
    {"type": "Feature", "properties": {"institution": "Universidade Federal de Juiz de Fora - Campus GV"}, "geometry": {"type": "Point", "coordinates": [-41.94918019310874, -18.856339060042096]}},
```

**Review by tibroc — CHANGES_REQUESTED** [review]
Um BBB com tradição! Saudades de Minas…

Just a caveat of this map: could you just flip the coordinates?

**Comment by felcaetano** [comment]
> Um BBB com tradição! Saudades de Minas…
I cannot measure how grateful we are to BigBlueButton. It was already really useful for us before covid, imagine after it... 


> Just a caveat of this map: could you just flip the coordinates?
Sure, uai! 😁

**Pull request merged**